src/lando/main/tests/conftest.py

```diff
@@ -1,11 +1,16 @@
-import pathlib
 import subprocess
+from pathlib import Path
 
 import pytest
 
 
 @pytest.fixture
-def git_repo(tmp_path: pathlib.Path):
+def git_repo_seed(request) -> Path:
```

Comment:
Can you add a docstring here to explain what this is doing? Also, it doesn't look like `request` is being used.